data/data/aws/master/main.tf

```diff
@@ -74,7 +74,7 @@ resource "aws_instance" "master" {
 
   iam_instance_profile = "${aws_iam_instance_profile.master.name}"
   instance_type        = "${var.instance_type}"
-  subnet_id            = "${element(var.subnet_ids, count.index)}"
+  subnet_id            = "${var.az_to_subnet_id[var.availability_zones[count.index]]}"
```

Comment:
wouldn't this out of index when masters are more than availability_zones ?

Reply:
Oh, i see https://github.com/openshift/installer/pull/1121/files#diff-001b1e593355c674310fd39854ea35cdR3